yarn.lock

```diff
@@ -94,24 +90,14 @@ faye-websocket@0.9.3:
   dependencies:
     websocket-driver ">=0.5.1"
 
-firebase-admin@^4.1.3:
-  version "4.1.3"
-  resolved "https://registry.yarnpkg.com/firebase-admin/-/firebase-admin-4.1.3.tgz#7203b0914172dde224b1b20fbcb8c7923d548778"
+firebase-admin@^4.2.1:
+  version "4.2.1"
+  resolved "https://registry.yarnpkg.com/firebase-admin/-/firebase-admin-4.2.1.tgz#91794bfc214ee21decc765d308411166a05c0b20"
   dependencies:
     "@types/jsonwebtoken" "^7.1.33"
     faye-websocket "0.9.3"
     jsonwebtoken "7.1.9"
 
-firebase@^3.7.2:
```

Comment:
I guess this is gone because of the change in https://github.com/algolia/algolia-firebase-nodejs/commit/dc98f1b7586c43f432c1e6b26e432e2119e0d88a didn’t run `yarn`